README.rst

```diff
@@ -16,11 +16,18 @@ Documentation (albeit still vastly incomplete as it is a work in progress) can b
 Requirements
 ============
 
+- Linux : 
```

Comment:
Why do you use a bullet point here ? It looks weird.

![image](https://user-images.githubusercontent.com/404665/183396891-009bb20d-0fc9-4455-b621-a6755915851c.png)